scripts/generators/enum_string_helper_generator.py

```diff
@@ -31,20 +31,6 @@ def generate(self):
 #include <vulkan/vulkan.h>
 ''')
 
-        # TODO - this should be moved into different generated util file
```

Comment:
... can take action now to move this in VVL so it doesn't break when we switch over to VUL